node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1030,7 +1030,8 @@ mod tests {
         chainspec.core_config.minimum_era_height = minimum_era_height;
 
         // `merkle_tree_hash_activation` can be chosen arbitrarily
-        let merkle_tree_hash_activation = EraId::from(rng.gen::<u64>());
+        let log_normal = LogNormal::new(2.5, 3.7).unwrap();
```

Comment:
Ok, updated in https://github.com/casper-network/casper-node/pull/2531/commits/718213235b419e3e44699eaf5682ee1e1fbff8bb.

I'm usually cool about additional `dev-dependencies`, especially the de-facto standard ones. I could achieve what I imagined with the already included `proptest` but thought it would be overkill.

Dependencies aside, what worries me a bit more is testing the system with input data that covers a tiny fraction of possible values (`1..=10` out of `u64::MAX`). Anyway, changes to that, if needed at all, should not be a part of this PR though, hence the revert.